src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -150,6 +150,7 @@ class AirspeedModule : public ModuleBase<AirspeedModule>, public ModuleParams,
 	float _ground_minus_wind_TAS{0.0f}; /**< true airspeed from groundspeed minus windspeed */
 	float _ground_minus_wind_CAS{0.0f}; /**< calibrated airspeed from groundspeed minus windspeed */
 	bool _armed_prev{false};
+	bool _armed{false};
```

Comment:
The same information as in _armed can be read with `_vehicle_status.arming_state == vehicle_status_s::ARMING_STATE_ARMED `correct?
I wouldn't make a global variable out of it. That adds an extra state, and the more states the more complexity often.

Reply:
resolved with [6d9d261](https://github.com/PX4/PX4-Autopilot/pull/19173/commits/6d9d261e78a0ebc0019ee05f3bc2320c302fb4b1)